src/app.js

```diff
@@ -112,60 +122,61 @@ class App extends PureComponent {
 
     return (
       <Router history={history}>
-        <div>
-          <AjaxLoader show={loading} size={120} />
-          <Route
-            path={['/auth/logout', '/auth/callback', '/error', '/404', '/app/start', '/app/profile']}
-            children={({ match }) => (
-              <Header language={language} profilePic={profile_pic} initLogOut={initLogOut} waitForApi={!match} />
-            )}
-          />
-
-          <Switch>
-            <LogOutCallbackRoute path="/auth/logout" doLogout={doLogout} />
-            <AuthorizationCallbackRoute
+        <SelectionPlanContext.Provider value={{selectionPlanCtx, setSelectionPlanCtx: this.setSelectionPlanCtx}}>
+
+          <div>
+            <AjaxLoader show={loading} size={120}/>
+            <Route
+              path={['/auth/logout', '/auth/callback', '/error', '/404', '/app/start', '/app/profile']}
```

Comment:
is this array correct for property path
on console i got this warning
Warning: Failed prop type: Invalid prop `path` of type `array` supplied to `Route`, expected `string`.
    in Route (created by App)
    in App (created by Connect(App))
    in Connect(App)
    in PersistGate
    in Provider